apps/safari/safari.py

```diff
@@ -6,7 +6,8 @@
 apps = mod.apps
 mod.apps.safari = """
 os: mac
-and app.bundle: com.apple.Safari
+app.bundle: com.apple.Safari
+app.bundle: com.apple.SafariTechnologyPreview
 """
 
 ctx.matches = r"""
```

Comment:
Line 18/19 below might need to be updated. It's only called on line 31/32.

I can't figure out where `address()` is actually used. E.g. when I do "copy url" when from STP when Safari is also running and on a different page, I get the right address. @nriley, do you have ideas how I can exercise the functionality of address()? Sorry for dumb question.

Reply:
Sure, it gets used when matching a website.

Reply:
Got it, thanks. Updated this PR to make that work in Safari or STP.